%YAML:1.0

# input options
online: 1           # input from serial port(1) or file(0)
## if `online == 1` then following options may be set
input_serial_port: "/dev/ttyACM1"
#input_serial_port: "/dev/ubloxF9P"
serial_baud_rate: 921600
input_rtcm: 0       # whether input RTCM data to receiver for RTK solution
rtcm_tcp_port: 3503
config_receiver_at_start: 0     # whether to config receiver during booting, ** NOT SUPPORT YET **
receiver_config_filepath: "~/catkin_ws/src/ublox_driver/config/ublox_rcv_config.yaml"
## if `online == 0` then following options may be set, `serial_baud_rate` is used for speed control
ubx_filepath: "~/tmp/ublox_driver_test/2020_11_30_7_14_33.ubx"
rtk_correction_ecef:      # apply RTK correction in case of the position of base station reported in RTCM is biased
    rows: 3
    cols: 1
    dt: d
    data: [ 0, 0, 0 ]

# output options
to_ros: 1           # publish as ros topic
to_file: 0          # dump data to file
dump_dir: "~/tmp/ublox_driver_test/"      # dump data to this directory, if applicable
to_serial: 0        # output data to serial port, for debugging purpose
output_serial_port: "/dev/ttyACM1"                  # dump data to this serial port, if applicable
#output_serial_port: "/dev/ubloxF9P"                  # dump data to this serial port, if applicable
